app/steps/ui/executors/contactdetails/index.js

```diff
@@ -50,8 +48,6 @@ class ExecutorContactDetails extends ValidationStep {
             executor.emailChanged = true;
```

Comment:
As per our conversation during a test run when changing the Executor email (after notifying) I wasn't able to then renotify the executor. The notification screen didn't show and the "Check your answers and make your legal declaration" was still selectable so the user would end up going around in a loop.